README.md

```diff
@@ -205,6 +207,31 @@ describe( "duck reducer", function( ) {
 } );
 ```
 
+### Typescript
+Your interface and type definitions must also be a part of your duck interface. I consider keeping them with the source generally a good practice. In order to allow import destructuring your duck must export them as named exports from index.ts. I usually find myself having interface definitions in three places: actions.ts (action interfaces), reducers.ts (state interfaces) and selectors.ts (selector interfaces). 
```

Comment:
a few comments on this:
- I would write it in a more impersonal style (no I) since it should open to changes from anyone in the community, so it's not a single opinionated document.
- I'm not very used to TypeScript, but can't all the interfaces/types be stored in a single separate file like the redux action types?

Reply:
- Sure, I can re-write it.
- For me, keeping all interfaces in a separate file doesn't make that much sense. It would add unnecessary imports inside the duck (basically these interfaces will be imported to every file). Doesn't it make sense to keep action related interfaces with actions?

Reply:
ok, makes sense. maybe you can do a quick example project with TS and this folder structure and link it in this repo? These's a section with a couple of example links

Reply:
sure, np. but it have to wait by the end of the week